x-pack/plugins/session_view/public/components/SessionViewPage/index.tsx

```diff
@@ -14,12 +14,105 @@ import { CoreStart } from '../../../../../../src/core/public';
 import { RECENT_SESSION_ROUTE, BASE_PATH } from '../../../common/constants';
 
 import { SessionView } from '../SessionView';
-import { ProcessEvent } from '../../../common/types/process_tree';
+import { ProcessEvent, EventKind, EventAction } from '../../../common/types/process_tree';
 
 interface RecentSessionResults {
   hits: any[];
 }
 
+const jumpToEvent: ProcessEvent = {
```

Comment:
this is temporary and only works with my locally generated data. will remove this once i get some unit tests written.